ext/gd/tests/avif_decode_encode.phpt

```diff
@@ -0,0 +1,57 @@
+--TEST--
+avif decoding/encoding tests
+--SKIPIF--
+<?php
+    if (!extension_loaded('gd')) {
+        die("skip gd extension not available.");
+    }
```

Comment:
Note that tests should no longer check whether extensions are loaded, but rather specify the required extensions in an `--EXTENSIONS--` section; see PR #6994.

Reply:
got it - thanks!

Reply:
Looks like I can specify `gd` in `--EXTENSIONS--`, but I still need to check for specific image type support. I'll need to rebase off the latest master to make sure I did this right....